cluster/addons/metrics-server/metrics-server-deployment.yaml

```diff
@@ -10,28 +10,31 @@ metadata:
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: metrics-server
```

Comment:
Hmm, why do this? This a Deployment, not a ReplicationController...

Reply:
I believe it's to appease the arcane rules followed by the addon manager (it will only apply changes when it sees metadata changes, and it's easy to change the name).

Reply:
TBH I don't know what is the current logic of addon-manager so I just followed Heapster pattern.

Reply:
Ok